portfolio/src/main/webapp/learn.js

```diff
@@ -0,0 +1,54 @@
+// More API functions here:
```

Comment:
Can you add some more comments hhere about what you expect this learn.js file to do?

Reply:
i don't think you added comments here